pyproject.toml

```diff
@@ -54,9 +54,7 @@ pydocstyle = "*"
 pylint = "*"
 pylint-django = "*"
 pylint-nautobot = "*"
-python-dotenv = "^1.0.0"
 yamllint = "*"
-toml = "*"
```

Comment:
Why is this removed? Is toml in the stdlib now?

Reply:
Removed